sale_delivery_state/models/sale_order_line.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2023 Manuel Regidor <manuel.regidor@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    skip_sale_delivery_state = fields.Boolean(
+        string="Skip Delivery State",
+        compute="_compute_skip_sale_delivery_state",
+        store=True,
+        readonly=False,
+    )
+
+    @api.depends("company_id", "product_id")
+    def _compute_skip_sale_delivery_state(self):
+        for sel in self:
```

Comment:
I do usually prefer to use for record in self: or for line in self:

Reply:
done